### ISSUE TITLE
Bug Report - CTD Walking Towards Nightgate Inn

### ISSUE THREAD

**Issue by Smeags777** [ForgottenGlory/Living-Skyrim-3#300]
**LS Version**
LS 3.1.1

**Describe the bug**
CTD upon heading downhill from the mountain pass between Alftand and Nightgate Inn. Screenshot provided shows rough location.

https://cdn.discordapp.com/attachments/690616522032545812/829621165316112424/CTD_Location.png

**To Reproduce**
Steps to reproduce the behavior: Head downhill (south) toward Nightgate Inn from the pass. 

**Expected behavior**
Travelling downhill without crashing.

**Screenshots**
Attached above.

**Additional context**
CRASH REPORT - 3 different CTDs in the same location.
[Crash_2021_4_8_17-41-2.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6277035/Crash_2021_4_8_17-41-2.txt)
[Crash_2021_4_8_17-24-33.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6277039/Crash_2021_4_8_17-24-33.txt)
[Crash_2021_4_8_17-28-2.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6277040/Crash_2021_4_8_17-28-2.txt)

**Comment by Fuzzydemon** [comment]
I get the same issue just below Nightgate Inn. More specifically the woods to the left of Bronze Water Cave.
[Crash_2021_4_10_15-49-27.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6290027/Crash_2021_4_10_15-49-27.txt)
[Crash_2021_4_10_16-14-43.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6290028/Crash_2021_4_10_16-14-43.txt)

**Comment by bananashot** [comment]
The same issue, it seems the whole region near Nightgate Inn is affected. Examples:

CTD during the The Jagged Crown quest (Civil War) + just a new game and start in Nightgate Inn CTD on exit from the inn.
[Crash_2021_4_12_10-33-57.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6294779/Crash_2021_4_12_10-33-57.txt)

**Comment by Serenoxe** [comment]
I seem to be having the same issue. Started when going toward the ruins for the jagged crown quest from Whiterun. Happened again when headed toward Nightgate Inn from Fort Fellhammer. 
[Crash_2021_4_13_21-9-35.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6307565/Crash_2021_4_13_21-9-35.txt)

**Comment by ZhamelSeh** [comment]
I have been having the same issue, CTD anytime I get near Nightgate Inn.

Through some testing I have isolated the issue to Miscellaneous Merged.esp, something in here is causing it. I have "fixed" the CTD by using SSEEditQuickAutoClean on this .esp, I know this is not fixing it properly, and it may cause issues elsewhere, but I have been able to get into the Inn, and have not had a CTD for about 6 hours or so while doing quests and stuff in/around the Inn. I also made a backup of the .esp as a just in case, along with saves :)

So, maybe this information will help in isolating the issue and help with a proper fix for everyone.
